plugins/module_utils/job.py

```diff
@@ -371,7 +371,6 @@ def _get_job_status(job_id="*", owner="*", job_name="*", dd_name=None, dd_scan=T
             job["queue_position"] = entry.queue_position
             job["program_name"] = entry.program_name
             job["class"] = ""
-            job["content_type"] = ""
```

Comment:
So content type was always empty before? or how was it retrieved?

Reply:
I don't know if it was always empty, but it was at least while I worked on this item and it was repeating the information I was adding from ZOAU.